lib/testing/src/lib/process-services/pages/process-list.page.ts

```diff
@@ -45,4 +45,8 @@ export class ProcessListPage {
     checkContentIsNotDisplayedByColumn(column: string, processName: string): Promise<void> {
         return this.dataTable.checkContentIsNotDisplayed(column, processName);
     }
+
+    async selectRow(processName): Promise<void> {
```

Comment:
missing type in the input parameter

Reply:
Can we rename it in 
```
async selectRowByName(processName: string)
```

Reply:
@mauriziovitale  should I also change the name of the same method from `task-list.page.ts` in order to have consistency?